### ISSUE TITLE
Smart-Marker: cursor stops AFTER the marker. Not ON the marker

### ISSUE THREAD

**Issue by tormyvancool** [reaper-oss/sws#1864]
Reapere 7.15
SWS 2.12.1

by using the smart-markers to investigate an issue an user highlighted, I noticed that
if I use smartmarkers,the cursor executes them AFTER (well after) the marker and not ON the marker

Here the !1008 (Pause). Note that the cursor came to the stopAFTER the marker and not ON the marker

Underneath another screenshot.
I set the marker up with the action 10157 it means "insert marker"

When the cursor passed, created the marker 4 ... see the distance ...

![image](https://github.com/reaper-oss/sws/assets/32305535/d86eae9b-2188-47b5-8f98-3fba8c7bb379)
![image](https://github.com/reaper-oss/sws/assets/32305535/1b1bdc73-a8bc-4d14-ba6b-38d36447d1c7)

**Comment by cfillion** [comment]
Action makers are not sample accurate. SWS checks 30 times per second (nominal, can be slower!) if playback crossed an action marker.

There is no way for a REAPER extension or ReaScript to schedule something to occur at an exact time position.

**Comment by tormyvancool** [comment]
> Action makers are not sample accurate. SWS checks 30 times per second (nominal, can be slower!) if playback crossed an action marker.
> 
> There is no way for a REAPER extension or ReaScript to schedule something to occur at an exact time position.
> 
> > SWS 2.12.1
> 
> By the way this version is 3 years out of date. The latest is 2.14 which is available from ReaPack or [here](https://www.sws-extension.org/download/pre-release/).

Ok so we can't get better but the difference is very small (negligible) then.

> Action makers are not sample accurate. SWS checks 30 times per second (nominal, can be slower!) if playback crossed an action marker.
> 
> There is no way for a REAPER extension or ReaScript to schedule something to occur at an exact time position.
> 
> > SWS 2.12.1
> 
> By the way this version is 3 years out of date. The latest is 2.14 which is available from ReaPack or [here](https://www.sws-extension.org/download/pre-release/).

Another thing: I did install SWS via ReaPack when it was available this feature, to try it. I update ReaPack several times in these few dasy: Why didn't it update also SWS?

**Comment by cfillion** [comment]
> Why didn't it update also SWS?

Only v2.14 is in ReaPack, so if you installed it from there (ReaPack > Browse packages > SWS > Install), you already got 2.14.

**Comment by ZipSnipe** [comment]
Its a shame this can't be fixed. I had a string of videos in Reaper using the pause markers but as it went down the timeline it gets more out of sync. Oh and I was using it also to control FLstudio which was controlling DMX lighting. The first three videos would play perfect the lights we in sync but as we got to the 4th video and on, it became more out of sync...sadly. Now Flstudio has pause markers, they do stop right on the money but they also will occasionally skip the marker not good for a live situation.  I ended up removing the pause markers from my set. Not what I wanted to do but was my only choice as of now.
